clinica/converters/ukb_to_bids/_utils.py

```diff
@@ -349,9 +352,38 @@ def write_bids(
             )
             if metadata["modality_num"] == "20217":
                 _import_event_tsv(bids_path=to)
+
+        bids_full_path = Path(bids_full_path)
+        scans_filepath = (
+            to
+            / str(metadata.participant_id)
+            / str(metadata.sessions)
+            / f"{metadata.participant_id}_{metadata.sessions}_scan.tsv"
```

Comment:
```suggestion
            / f"{metadata.participant_id}_{metadata.sessions}_scans.tsv"
```

See the BIDS specifications here: https://bids-specification.readthedocs.io/en/stable/modality-agnostic-files.html#scans-file